terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/test/resources/org/terasoluna/gfw/common/codelist/i18n/simpleI18nCodeList.xml

```diff
@@ -124,6 +163,47 @@
     <property name="fallbackTo" value="en" />
   </bean>
 
+  <bean id="CL_testSetColumns02" class="org.terasoluna.gfw.common.codelist.i18n.SimpleI18nCodeList">
+    <property name="columns">
+      <util:map>
+        <entry key="0">
+          <util:map>
+            <entry key="en_US" value="Sun." />
+          </util:map>
+        </entry>
+        <entry key="1">
+          <util:map>
+            <entry key="en_US" value="Mon." />
+          </util:map>
+        </entry>
+        <entry key="2">
+          <util:map>
+            <entry key="en_US" value="Tue." />
+          </util:map>
+        </entry>
+        <entry key="3">
+          <util:map>
+            <entry key="en_US" value="Wed." />
+          </util:map>
+        </entry>
+        <entry key="4">
+          <util:map>
+            <entry key="en_US" value="Thu." />
+          </util:map>
+        </entry>
+        <entry key="5">
+          <util:map>
+            <entry key="en_US" value="Fri." />
+          </util:map>
+        </entry>
+        <entry key="6">
+          <util:map>
+            <entry key="en_US" value="Sat." />
+          </util:map>
+        </entry>
+      </util:map>
+    </property>
+  </bean>
```

Comment:
- [ ] No test of  `<property name="fallbackTo">` variation.
https://github.com/terasolunaorg/terasoluna-gfw/blob/96a51b13a4561ed2b284f8e233fb179b3134b518/terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/main/java/org/terasoluna/gfw/common/codelist/i18n/SimpleI18nCodeList.java#L330-L333
https://github.com/terasolunaorg/terasoluna-gfw/blob/96a51b13a4561ed2b284f8e233fb179b3134b518/terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/main/java/org/terasoluna/gfw/common/codelist/i18n/SimpleI18nCodeList.java#L344-L346

Reply:
Modified by da7cfba